src/R/Components/Impl/Plots/IRPlotHistoryVisualComponent.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.R.Components.Plots.ViewModel;
+using Microsoft.R.Components.View;
+
+namespace Microsoft.R.Components.Plots {
+    public interface IRPlotHistoryVisualComponent : IVisualComponent {
+        IRPlotHistoryViewModel ViewModel { get; }
```

Comment:
`IVisualComponent` is a facade interface between UI and non-UI parts of component. ViewModel belongs to UI, so we shouldn't expose it to the VisualComponent. Of course, there is a `IWpfTextView`, which is exposed and is also a UI component, but that should also be refactored later.